nextflow.config

```diff
@@ -51,25 +56,28 @@ process.shell = ['/bin/bash', '-euo', 'pipefail']
 
 timeline {
   enabled = true
-  file = "${params.outdir}/pipeline_info/nf-core/mag_timeline.html"
+  file = "${params.tracedir}/pipeline_info/nf-core/mag_timeline.html"
```

Comment:
I think this line is better as it is currently in the template:
`  file = "${params.tracedir}/{{ cookiecutter.name.replace(' ', '-') }}_timeline.html"`
As it is now, I will end up with `results/pipeline_info/pipeline_info/nf-core/mag_trace.txt` as default.